crypto/keys/multisig/multisig.go

```diff
@@ -0,0 +1,128 @@
+package multisig
+
+import (
+	fmt "fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	crypto "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	tmcrypto "github.com/tendermint/tendermint/crypto"
+
+	"github.com/cosmos/cosmos-sdk/crypto/types/multisig"
+)
+
+var cdc = codec.NewProtoCodec(types.NewInterfaceRegistry())
```

Comment:
We need to use Amino here, as this codec will determine the Bytes, and hence the Address, of the multisig.